components/kyma-environment-broker/internal/avs/model.go

```diff
@@ -116,14 +116,35 @@ func newBasicEvaluationCreateRequest(operation internal.ProvisioningOperation, e
 	}, nil
 }
 
-func generateNameAndDescription(globalAccountId string, subAccountId string, name string, beType string) (string, string) {
-	beName := fmt.Sprintf("K8S-AZR-Kyma-%s-%s-%s", beType, subAccountId, name)
-	beDescription := fmt.Sprintf("SKR instance Name: %s, Global Account: %s, Subaccount: %s",
-		name, globalAccountId, subAccountId)
+func generateNameAndDescription(operation internal.ProvisioningOperation, beType string) (string, string) {
+	globalAccountID := operation.ProvisioningParameters.ErsContext.GlobalAccountID
+	subAccountID := operation.ProvisioningParameters.ErsContext.SubAccountID
+	name := operation.ProvisioningParameters.Parameters.Name
+	shootName := operation.InstanceDetails.ShootName
+	beName := fmt.Sprintf("K8S-%s-Kyma-%s-%s-%s", providerCodeByPlan(operation.ProvisioningParameters.PlanID), beType, subAccountID, name)
+	beDescription := fmt.Sprintf("SKR instance Name: %s, Global Account: %s, Subaccount: %s, Shoot: %s",
+		name, globalAccountID, subAccountID, shootName)
 
 	return truncateString(beName, 80), truncateString(beDescription, 255)
 }
 
+func providerCodeByPlan(planID string) string {
+	switch planID {
+	case broker.AWSPlanID:
+		return "AWS"
+	case broker.GCPPlanID:
+		return "GCP"
+	case broker.AzurePlanID, broker.AzureLitePlanID, broker.AzureHAPlanID:
+		return "AZR"
+	case broker.TrialPlanID, broker.FreemiumPlanID:
```

Comment:
freemium could be AWS or Azure

Reply:
exactly, I think we should do first https://github.tools.sap/kyma/backlog/issues/1473 and then adjust and merge this PR.